ceph/utils.py

```diff
@@ -165,6 +165,33 @@ def volume_cleanup(volume, osp_cred):
                     return 1
 
 
+# Added this function to cleanup stale volumes with status deleting, available, error, unknown, reserved
```

Comment:
can you please this as doc string within definition.
```
def cleanup_ceph_vols(...):
'''
Added this function to cleanup stale volumes with status deleting, available, error, unknown, reserved
'''

Reply:
Updated the comment to docstring.